### ISSUE TITLE
[FEATURE REQUEST] Allow Bridge app to open SmartTubeNext itself

### ISSUE THREAD

**Issue by PeterNjeim** [yuliskov/SmartTubeNext#1727]
I understand what the purpose of the Bridge app is, it's to allow voice searches, this works as intended. On the Google Chromecast with Google TV (Google Sabrina) remote, there is a YouTube button that is supposed to open the YouTube app. If YouTube is uninstalled, it simply opens the Google Play Store page of YouTube. Even with the Bridge app installed, it still takes me to the Google Play Store page, but this time it shows an "uninstall" button (obviously since it has the same app ID). Maybe a nice feature would be to allow the Bridge app to open the SmartTubeNext app itself, so that the YouTube button on the remote would work as well. Someone else asked for this feature a while ago but closed their own issue without explanation: https://github.com/yuliskov/SmartTubeNext/issues/1266

Don't worry though, there is a workaround for it: https://github.com/yuliskov/SmartTubeNext/issues/1309, so it's not urgent at all.

**Comment by FloKTC** [comment]
i hope the devs see this post

**Comment by PeterNjeim** [comment]
@FloKTC I currently use the workaround by remapping my YouTube button on my remote to open SmartTubeNext, so I forgot I even made this issue lol. Did you try remapping the button? It works as expected

**Comment by FloKTC** [comment]
@PeterNjeim  yeah if done that a couple minutes ago , it would be nice to have the option to replace the normal youtube app with smartube  but i think we need a jailbreak for that on chromecast

**Comment by PeterNjeim** [comment]
Chromecast meaning... the Chromecast with Google TV? That's the one I use, I fully replaced the regular YouTube with SmartTubeNext. I use an app called Key Remapper to remap the YouTube button to open SmartTubeNext, and the Bridge app makes the voice search work from the home screen. I uninstalled the default YouTube app as well.

**Comment by FloKTC** [comment]
yeah the chromecast with google tv , i have still the orginal youtube app because of the recommendations (and continue to watch a video )   on the home screen , and sometimes i cant cast to smartube  thats why i also have the youtube app
